app/templates/package.json

```diff
@@ -23,27 +23,29 @@
   },
   "homepage": "https://github.com/<%= user %>/<%= repo %>",
   "devDependencies": {
-    "babel-core": "^5.2.17",
-    "babel-eslint": "^4.0.5",
-    "babel-loader": "^5.3.2",
-    "chai": "^3.2.0",
+    "babel-core": "^6.3.26",
```

Comment:
@jmeas it's slated to be deprecated. It shouldn't bloat the install as long as compatible versions are depended on because the 2 packages are currently mutually dependent anyway, with `babel-core/register` just re-exporting `babel-register`.